### PR TITLE
3) Register notification aftermath

### LINKED ISSUE #15: Connect functions together - `registerNotification`
Function [`registerNotification`](https://github.com/covid19cz/erouska-backend/wiki/Firebase-Functions#registernotification) internally calls some other functions (such as [`increaseIpAttemptsCount`](https://github.com/covid19cz/erouska-backend/wiki/Firebase-Functions#increaseipattemptscount).  
Wire them together, when they are all ready.

### PR DESCRIPTION
**Pull request by jendakol**
Closes #15

### CODE REVIEW

File: internal/constants/constants.go [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package constants
 
+//ProjectID GCP project ID
+const ProjectID = "erouska-key-server-dev"
```
Comment (marekaf) [inline internal/constants/constants.go:4]: this should not be hardcoded, I'll make an issue for that and fix it myself:)

File: internal/functions/registernotification/register-notification-aftermath.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	if err != nil {
+		logger.Warnf("Cannot handle register notification aftermath due to unknown error: %+v", err.Error())
+		return err
+	}
+
+	logger.Debugf("Daily count for %v: %v", payload.Ehrid, finalDailyCount)
+
+	// Step 2. Check if daily state is not too high
+
+	var thresholdsOK = finalDailyCount == 1
```
Comment (marekaf) [inline internal/functions/registernotification/register-notification-aftermath.go:85]: Alt Text

File: builders/deploy.yaml [hunk 3]
```diff
@@ -85,9 +52,9 @@ steps:
   args:
     - functions
     - deploy
-    - ProvideVerificationCode
+    - RegisterNotificationAfterMath
     - --source=.
-    - --trigger-http
+    - --trigger-topic=notification-registered
```
Comment (marekaf) [inline builders/deploy.yaml:57]: this topic is created by hand? I'll add it to terraform in another PR:)

**Comment by marekaf** [comment]
lgtm!

File: internal/constants/constants.go (hunk 1 above)
Comment (jendakol) [reply internal/constants/constants.go:4]: I know. It's the same as was/is `FirebaseUrl`.

File: builders/deploy.yaml (hunk 3 above)
Comment (jendakol) [reply builders/deploy.yaml:57]: It's created automa[tg]ically when it doesn't exist.

File: internal/functions/registernotification/register-notification-aftermath.go (hunk 2 above)
Comment (jendakol) [reply internal/functions/registernotification/register-notification-aftermath.go:85]: Yeah, I know, but it's exactly what the [requirement](https://github.com/covid19cz/erouska-backend/wiki/Firebase-Functions#registernotificationaftermath) was :smile:

File: builders/deploy.yaml (hunk 3 above)
Comment (marekaf) [reply builders/deploy.yaml:57]: aha!

File: internal/functions/registernotification/register-notification-aftermath.go (hunk 2 above)
Comment (marekaf) [reply internal/functions/registernotification/register-notification-aftermath.go:85]: I'm talking about the confusing syntax `something = count == n` :D

Comment (marekaf) [reply internal/functions/registernotification/register-notification-aftermath.go:85]: and I wanted to try gifs in comments :D

**Pull request merged**